CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ if(MSVC OR WIN32 OR CYGWIN OR APPLE)
 else()
     option(GINKGO_BUILD_HWLOC "Build Ginkgo with HWLOC. Default is ON. If a system HWLOC is not found, then we try to build it ourselves. Switch this OFF to disable HWLOC." ON)
 endif()
+option(GINKGO_WITH_PAPI_SDE "Build Ginkgo with PAPI SDE. Default is OFF." OFF)
```

Comment:
We should have clear naming scheme - `GINKGO_HAVE_*` indicates the presence of (mostly autodetected) libraries that cause part of Ginkgo to be enabled via `ifdef`. `GINKGO_BUILD_*` means actively enabling a component. What does WITH stand for?

Reply:
See Marcel's comment above. https://github.com/ginkgo-project/ginkgo/pull/1321#discussion_r1159906445
I generally am fine with the `BUILD` and `HAVE` combo as well.

Reply:
I ended up going back to the `BUILD`/`HAVE` duo for consistency with our other options.